core/src/test/java/org/apache/jsieve/HeaderTest.java

```diff
@@ -57,6 +57,21 @@ public void testHeaderIsTrue() {
     /**
      * Test for Test 'header'
      */
+    @Test(expected = ThrowTestException.TestException.class)
+    public void testFoldedEncodedHeader() throws Exception {
```

Comment:
BTW you are commenting outdated commit of an other pull request. Sorry. https://github.com/linagora/james-jsieve/pull/9